packages/block-library/src/button/style.scss

```
@@ -50,15 +50,15 @@ $blocks-block__margin: 0.5em;
 	}
 
 	&.wp-block-button__width-25 {
-		width: calc(25% - (var(--wp--style--block-gap, #{$blocks-block__margin}) * 0.75));
+		width: calc(25% - (var(--wp--style--block-column-ap, #{$blocks-block__margin}) * 0.75));
```

Just noticed a tiny typo while skimming this PR 🙂
```suggestion
		width: calc(25% - (var(--wp--style--block-column-gap, #{$blocks-block__margin}) * 0.75));
```